distribution/lib/Standard/Base/0.0.0-dev/src/Runtime/Lazy_Ref.enso

```diff
@@ -0,0 +1,9 @@
+## PRIVATE
```

Comment:
Module can't be private as used in Table_Tests.

Reply:
Aren't tests supposed to run with `--disable-private-check`?

Reply:
No, at least not right now. We need to design a sensible approach to that.

If we run all tests with `--disable-private-check` we will not catch regressions where one library uses another's `private` entity. In tests with `--disable-private-check` such usage will succeed, but then in production it would fail.

It's not an imagined scenario - while refactoring I already encountered it numerous times where I made "too many things private" and things stopped working. If tests are ran with this flag without some further design, we will risk breaking too easily.

IMO we could add a separate project like `Internals_Tests`, or for each project have e.g. `Table_Internals_Tests`. Such test project could run with `--disable-private-check`. But the main suite cannot, at least not in the current setup.